src/js/view/view-model.js

```diff
@@ -0,0 +1,115 @@
+import SimpleModel from 'model/simplemodel';
+
+const INITIAL_VIEW_STATE = {
+    viewSetup: false,
```

Comment:
Can we alphabetize these?

Reply:
I'm going to remove these and just keep all state in the model. Storing these attributes in the view-model requires that the view code only write to the view-model properties and not be able to write to the model, which it currently does - "width", "height"...